packages/babel-helpers/src/helpers.js

```diff
@@ -377,6 +377,16 @@ helpers.inherits = template(`
   })
 `);
 
+helpers.inheritsLoose = template(`
+  (function (subClass, superClass) {
+    subClass.prototype = Object.create(superClass && superClass.prototype);
+    subClass.prototype.constructor = subClass;
+    if (superClass) {
```

Comment:
Ok we can just assume in loose mode that the superClass is non-null and has a prototype to get

```js
  (function (subClass, superClass) {
    subClass.prototype = Object.create(superClass.prototype);
    subClass.prototype.constructor = subClass;
    subClass.__proto__ = superClass;
  })
```